globals.go

```diff
@@ -832,6 +844,10 @@ func IsGitTagRequester(requester string) bool {
 	return requester == GitTagRequester
 }
 
+func IsCommitQueueRequester(requester string) bool {
+	return requester == MergeTestRequester
```

Comment:
Can we replace other instances of `t.Requester == evergreen.MergeTestRequester` with this?